README.md

```diff
@@ -1 +1,23 @@
-# SeedAvatarStudio
\ No newline at end of file
+# SeedAvatarStudio
+
+
+
+## Setup
+
+The recommended Unity editor version is 2021.3.30f1c1, but other editor may likely work.
```

Comment:
No need to include the required version in README I think, since the code file `ProjectSettings/ProjectVersion.txt` is a perfect location to maintain and display the current version info. Otherwise, you will need to update READE me every time you upgrade the Unity version.

If you want to say something about the version number, it'd be ideal to include a link to the code `ProjectSettings/ProjectVersion.txt`.